src/status_im/utils/utils.cljs

```diff
@@ -84,14 +88,21 @@
       (str/trim)))
 
 (defn first-index
-  [cond coll]
```

Comment:
I agree, I just renamed `cond` to `cond?` to avoid the nameclash with core clojure function, but pred is even better.